app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter4/conditionwatchers/ConditionWatchers.kt

```diff
@@ -213,4 +243,17 @@ object ConditionWatchers {
         })
         return interaction
     }
+
+    fun isDrawerNotDisplayed(): Boolean {
+        try {
```

Comment:
In Kotlin instead of: 
```
try {
    ...
    return false
} catch {
    return true
}
```
You can use this: 
```
return try {
    ...
    false
} catch {
    true
}
```